tests/python/relay/aot/test_c_device_api.py

```diff
@@ -136,44 +136,29 @@ def test_device_api_hooks_unpacked_api(device_api_main_func):
 
     # Activate Device
     assert (
-        str(main_func.body[0][0].value)
-        == "@tir.call_extern("
-        + '"TVMDeviceEthosUActivate",'
-        + " device_context_ethos_u: handle,"
-        + " dtype=int32)"
+        str(main_func.body[0])
+        == "tir.call_extern(" + '"TVMDeviceEthosUActivate",' + " device_context_ethos_u)\n"
     )
     # Open Device
+    print(str(main_func.body[1]))
```

Comment:
Is this meant to still be here? If so it would make sense to print the entire `body` before any asserts?

Reply:
You got me!. Removed.